docs/photoniq/prerendering/best-practices-prerendering.md

```diff
@@ -33,31 +29,8 @@ Customize your origin server to detect user-agent strings. This enables more gra
 
 Examine the performance metrics generated by the prerendering service. These metrics provide valuable insights into user experience and identify areas for further optimization. Make a habit of checking these metrics at regular intervals to stay ahead of any issues.
 
+For more information about monitoring Prerendering metrics, refer to [Monitor Prerendering](monitor-prerendering).
+
 ## Manage Cache on Your Origin Server
```

Comment:
I think we should also remove this section. It looks like it does not add any value or does not provide any action to be done by the users.